src/coreclr/jit/fgprofile.cpp

```diff
@@ -4446,10 +4446,7 @@ bool Compiler::fgComputeCalledCount(weight_t returnWeight)
     {
         // Skip past any/all BBF_INTERNAL blocks that may have been added before the first real IL block.
         //
-        while (firstILBlock->HasFlag(BBF_INTERNAL))
-        {
-            firstILBlock = firstILBlock->Next();
-        }
+        firstILBlock = fgGetFirstILBlock();
```

Comment:
I want to submit a change to compute `fgCalledCount` earlier before I merge this PR. `fgGetFirstILBlock` added by this PR will follow `BBJ_ALWAYS` to find the first IL block, but it seems dangerous to do that this late (after morph).

Reply:
I decided to revert this back to what it was  to get this PR in quicker. Can look at cleaning this up separately.